docs/local-setup.md

```diff
@@ -0,0 +1,46 @@
+# Local Setup
+
+## DDEV
+
+The SNAC Laravel application uses DDEV for local development.
+
+Follow the [getting started guide](https://ddev.com/get-started/) to install DDEV on your machine.
+
+### Running the server
+
+Once DDEV is installed, run the following command in the root of this project.
+
+```sh
+ddev start
+```
+
+#### IMPORTANT
+This application shares its database with the snac-server application and snac-server must be running and have an imported database before this application will work.
```

Comment:
I think I was looking for something like this at the top of the document, and when I didn't see it, I assumed this was what I needed to do first. Maybe put this at the top, and link to the `snac` repo to set that up first?

I definitely overlooked this though.